test/functional/tests/cli/test_seq_cutoff.py

```diff
@@ -0,0 +1,282 @@
+def test_seq_cutoff_set_get_policy_cache(policy):
+    """
+    title: Sequential cut-off policy set/get test for cache
+    description: |
+      Test if CAS is setting proper sequential cut-off policy for whole cache and
+      returns previously set value
+    pass_criteria:
+      - Sequential cut-off policy obtained from get-param command for each of 3 cores must be the
+        same as the one used in set-param command for cache
+    """
+    with TestRun.step("Test prepare (start cache and add 3 cores)"):
+        cache, cores = prepare(cores_count=3)
+
+    with TestRun.step(f"Setting sequential cut off policy mode {policy} for cache"):
+        cache.set_seq_cutoff_policy(policy)
+
+        for i in TestRun.iteration(range(0, len(cores))):
+            with TestRun.step(f"Check if proper sequential cut off policy was set for core "
+                              f"number {i}"):
+                if cores[i].get_seq_cut_off_policy() != policy:
+                    TestRun.fail(f"Wrong core sequential cut off policy: "
+                                 f"{cores[i].get_seq_cut_off_policy()} "
+                                 f"should be {policy}")
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_seq_cutoff_policy_load():
+    """
+    title: Sequential cut-off policy set/get test with cache load between
+    description: |
+      Set each possible policy for different core, stop cache, test if after cache load
+      sequential cut-off policy value previously set is being loaded correctly for each core.
+    pass_criteria:
+      - Sequential cut-off policy obtained from get-param command after cache load
+        must be the same as the one used in set-param command before cache stop
+      - Sequential cut-off policy loaded for the last core should be the default one
+"""
+    with TestRun.step(f"Test prepare (start cache and add {len(SeqCutOffPolicy) + 1} cores)"):
+        # Create as many cores as many possible policies including default one
+        cache, cores = prepare(cores_count=len(SeqCutOffPolicy) + 1)
+
+    with TestRun.step("Set all possible policies except the default one"):
+        policies = [policy for policy in SeqCutOffPolicy]
+        for i, core in TestRun.iteration(enumerate(cores[:-1])):
+            with TestRun.step(f"Setting cache sequential cut off policy mode to "
+                              f"{policies[i]} for the core number {i}"):
+                cores[i].set_seq_cutoff_policy(policies[i])
+
+    with TestRun.step("Stopping cache"):
+        cache.stop()
+
+    with TestRun.step("Loading cache"):
+        loaded_cache = casadm.load_cache(cache.cache_device)
+
+    with TestRun.step("Getting cores from loaded cache"):
+        cores = loaded_cache.get_core_devices()
+
+    with TestRun.step("Check if proper policies have been loaded"):
```

Comment:
You cannot have test step inside another test step. See GUIDELINES.md
Add description to instead. (Alternatively you could surround it with `with TestRun.group()`, but as `TestRun.iteration()` creates it's own group, that would be redundant).